ci/run.bash

```diff
@@ -50,21 +50,6 @@ target_cargo() {
 
 target_cargo build
 
-download_pkg_test() {
-  features=('--no-default-features' '--features' 'curl-backend,reqwest-native-tls')
```

Comment:
I believe this logic is still required at least for now: the `--features test` below won't suffice.

Reply:
Adapted this -- want to take another look?

Reply:
@djc This is looking good, thanks :)